### DISCUSSION TITLE
Supporting M2 models

### DISCUSSION THREAD

**Discussion by oasis791** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#749] [Q&A] [answered]
Hi!

First of all, thanks for the program.

I'm using Macbook Air M2 now, so I want to know when do you guys support this program for M2 model

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @oasis791,

We are currently working on adding support to M2 models and will let you know when it is officially supported.

Cheers,
Matthias
